build-farm/make-adopt-build-farm.sh

```diff
@@ -142,10 +142,10 @@ then
     # OpenJ9 only supports building jdk-11 with jdk-11
     JDK_BOOT_VERSION="11"
   fi
-  if [ "${JAVA_FEATURE_VERSION}" == "19" ]; then
+  if [ "${JAVA_FEATURE_VERSION}" == "17" ]; then
```

Comment:
This is not quite right, as now when building jdk-19 it will try and use a jdk-18 bootjdk.
The required jar --date option is only available on jdk-17u and jdk-19.
This if needs to be if 17 OR 19.